phpcs.xml

```diff
@@ -13,8 +15,17 @@
 	<exclude-pattern>*/node_modules/*</exclude-pattern>
 	<exclude-pattern>*/vendor/*</exclude-pattern>
 
+	<!-- Show progress, show the error codes for each message (source). -->
+	<arg value="ps"/>
+
+	<!-- Strip the filepaths down to the relevant bit. -->
+	<arg name="basepath" value="./"/>
+
+	<!-- Check up to 8 files simultaneously. -->
+	<arg name="parallel" value="8"/>
```

Comment:
In general all new lines should be include a empty space before `/>`.

Reply:
No it shouldn't. If you validate your XML according to the specs that will be flagged.

Reply:
@jrfnl all the other elements on this file does it.